src/backend/tools/py.CLI/caliopen_cli/commands/setup_storage.py

```diff
@@ -27,3 +27,32 @@ def setup_storage(settings=None):
         if hasattr(kls._model_class, 'pk'):
             # XXX find a better way to detect model from udt
             sync_table(kls._model_class)
+
+    # Add a default admin user
```

Comment:
in fact no create admin user automatically, just add in documentation to do docker run cli create_user -e admin ....

Reply:
OK. I need to refactor a bit APIv2 as — with current PR — it will not start if it can't find and admin user.